### ISSUE TITLE
lxqt-config-appearance will start/replace xsettingsd even if GTK theme management is disabled

### ISSUE THREAD

**Issue by M4he** [lxqt/lxqt-config#1049]
The LXQt appearance configuration dialog (`lxqt-config-appearance`) will start an `xsettingsd` instance even if its GTK theme management option is disabled.
This breaks any custom `xsettingsd` configurations a user might be running, because it will terminate those (only one `xsettingsd` instance can exist at a time).

##### Expected Behavior

If "Set GTK themes" is disabled in `lxqt-config-appearance`, opening up the appearance configuration dialog should *not* start a new `xsettingsd` instance or replace an existing one that wasn't started by `lxqt-config-appearance`.

##### Current Behavior

Even if "Set GTK themes" is disabled in `lxqt-config-appearance`, simply opening the appearance configuration dialog will start a new `xsettingsd` instance as:

```
/usr/bin/xsettingsd -c /tmp/lxqt-config-appearance.<random-string>
```
  
If an `xsettingsd` instance was already running (e.g. configured by the user) before opening the dialog, it is replaced by the dialog's.
When the appearance dialog is closed, its `xsettingsd` instance is also terminated¹. In case the user had a custom `xsettingsd` instance running before, it is not restored, effectively leaving the system running without an active instance¹.

¹ ~~note: is this actually a bug of its own? I don't see the purpose of a dedicated `xsettingsd` instance if it terminates after closing the dialog. It has to keep running to be effective.~~ (EDIT: see my comments below)

##### Possible Solution

a) check if the option "Set GTK themes" is set at the startup of `lxqt-config-appearance` and don't start an `xsettingsd` instance unless it's explecitly enabled

or

b) check if an `xsettingsd` instance is already running (which wasn't started by `lxqt-config-appearance`) and if so, do not replace it.

##### Steps to Reproduce (for bugs)

1. start a custom `xsettingsd` instance
    - e.g. `xsettingsd -c ~/.xsettingsd` (see [here](https://github.com/derat/xsettingsd/wiki/Installation#configuration) about configuration)
2. open up `lxqt-config-appearance`
3. observe your `xsettingsd` instance getting replaced (check your process manager)
4. close the appearance configuration dialog
5. observe that no `xsettingsd` instance remains running

##### Context

Users may want to use a custom `xsettingsd` instance (e.g. started via LXQt's autostart) containing custom tweaks and don't want LXQt to interfere.

LXQt's behavior is not obvious in this case. It does warn you about replacing `~/.gtkrc-2.0` and `~/.config/gtk-3.0/` stuff if you enable the GTK management but it doesn't notify/warn you that it replaced your `xsettingsd` instance.

##### System Information

* Distribution & Version: Debian 12 bookworm
* Kernel: 6.1.0
* Qt Version: 5.15.7
* liblxqt Version: 1.2.0
* Package version:

**Comment by Bluey26** [comment]
I can verify that Xsettings is not ran anymore

**Comment by stefonarch** [comment]
> I can verify that Xsettings is not ran anymore

On wayland only no?

**Comment by stefonarch** [comment]
>What is not clear to me is why xsettings is ran (in x11) if i have the option "set gtk settings theme" disabled. (PS:It was to make Qt changes on the fly,right?)

That's what this issue is about. Disabled xsettingsd for good under wayland in #1038

**Comment by Bluey26** [comment]
> A possible fix: #1051

Yes, that would fix the original poster issue.

In wayland is solved by removing xsettingsd completely.

**Comment by tsujan** [comment]
> If the set gtk theme then xsettingsd is launched (xsettingsd ... something about a tempfile) and the other xsettingsd process is killed (desired).

That's what the original coder did. I just made starting of `xsettingsd` dependent on the state of that check-box under X11.
